practice-01/index.html

```diff
@@ -115,7 +115,7 @@ <h5 class="card__title">Sales</h5>
                             class="product__pic">
                     </div>
                     <h3 class="card__text">The best cities for sales careers in 2022</h3>
-                    <h4 class="card__footer">20 Mar 2022, <span class="card__footer--gray"> Bill Holloway</span></h4>
+                    <h4 class="card__footer">20 Mar 2022, <span class="card__footer--gray">by Bill Holloway</span></h4>
```

Comment:
You can create a css class utility `text--gray` then use it in any text that have color `--west_chester-grey`. Btw, `grey` or `gray` plz pick only one.

Reply:
Don't foget update all similar cases. Ex: `text--white` , `text--dark`, .....